agent/engine/docker_image_manager_integ_test.go

```diff
@@ -230,9 +231,10 @@ func TestIntegImageCleanupThreshold(t *testing.T) {
 
 	// Verify Task is stopped
 	verifyTaskIsStopped(taskEvents, testTask)
+	testTask.SetSentStatus(api.TaskStopped)
 
 	// Allow Task cleanup to occur
-	time.Sleep(2 * time.Second)
+	time.Sleep(5 * time.Second)
```

Comment:
Event-based would be better, but we'd actually still want a timeout on the event.  Part of the tests here are ensuring that cleanup happens within the time that is expected.